tests/DefaultLogWriterTest.php

```diff
@@ -3,150 +3,129 @@
 namespace Spatie\HttpLogger\Test;
 
 use Illuminate\Http\UploadedFile;
-use Spatie\HttpLogger\DefaultLogWriter;
 
-class DefaultLogWriterTest extends TestCase
-{
-    /** @var \Spatie\HttpLogger\DefaultLogWriter */
-    protected $logger;
+use function PHPUnit\Framework\assertStringContainsString;
+use function PHPUnit\Framework\assertStringNotContainsString;
 
-    public function setUp(): void
-    {
-        parent::setup();
+use Spatie\HttpLogger\DefaultLogWriter;
 
-        $this->logger = new DefaultLogWriter();
-    }
+beforeEach(function () {
+    $this->logger = new DefaultLogWriter();
+});
 
-    /** @test */
-    public function it_logs_request_method_and_uri()
-    {
-        foreach (['post', 'put', 'patch', 'delete'] as $method) {
-            $request = $this->makeRequest($method, $this->uri);
+test('it logs request method and uri', function () {
+    foreach (['post', 'put', 'patch', 'delete'] as $method) {
+        $request = $this->makeRequest($method, $this->uri);
 
-            $this->logger->logRequest($request);
-        }
+        $this->logger->logRequest($request);
+    }
 
-        $log = $this->readLogFile();
+    $log = $this->readLogFile();
 
-        $this->assertStringContainsString("POST {$this->uri}", $log);
-        $this->assertStringContainsString("PUT {$this->uri}", $log);
-        $this->assertStringContainsString("PATCH {$this->uri}", $log);
-        $this->assertStringContainsString("DELETE {$this->uri}", $log);
-    }
+    assertStringContainsString("POST {$this->uri}", $log);
+    assertStringContainsString("PUT {$this->uri}", $log);
+    assertStringContainsString("PATCH {$this->uri}", $log);
+    assertStringContainsString("DELETE {$this->uri}", $log);
+});
 
-    /** @test */
-    public function it_will_log_the_body()
-    {
-        $request = $this->makeRequest('post', $this->uri, [
-            'name' => 'Name',
-        ]);
+test('it will log the body', function () {
```

Comment:
Another place where we could use the `it` function. Could you update this for all PRs you sent? (thanks for those PRs btw, nice job)